README.md

```diff
@@ -118,6 +121,21 @@ $ make start-test-dbs
 
 This will allow you to run the tests as normal in your IDE.
 
+### Updating the Schema
+
+- In the database repository update the schema.sql file
+- In SocialCareCaseViewerAPI/Infrastructure either update an existing class or create a new class for the schema changes
+- To test locally run `make restart-db`
+- To deploy changes to AWS:
+    - Go to AWS account (staging or prod)
+    - Go to Systems Manager
+    - Go to Session Manager
+    - Choose `RDS jump box-Platform APIs (new)` and click `Start Session`, this allows us to have a CLI into the instance hosting our database
+    - Connect to PostgreSQL `psql --host=social-care-db-production.c1rcth1f5uzc.eu-west-2.rds.amazonaws.com --port=5600 --username=<username> --password=<password> --dbname=social_care`
```

Comment:
Would it be a risk if we show the hostname publicly? 🤔

Reply:
It should be fine but I have amended it. No nasty hackers please

[hackerman](https://i.kym-cdn.com/entries/icons/original/000/021/807/ig9OoyenpxqdCQyABmOQBZDI0duHk2QZZmWg2Hxd4ro.jpg)